micrometer-core/src/test/java/io/micrometer/core/instrument/MeterRegistryTest.java

```diff
@@ -45,8 +45,8 @@ void idTransformingMeterFilter() {
         registry.config().meterFilter(MeterFilter.ignoreTags("k1"));
 
         registry.counter("my.counter", "k1", "v1");
-        assertThat(registry.find("my.counter").counter()).isPresent();
-        assertThat(registry.find("my.counter").tags("k1", "v1").counter()).isNotPresent();
+        registry.mustFind("my.counter").counter();
+        assertThat(registry.find("my.counter").tags("k1", "v1").counter()).isNull();
```

Comment:
Here you can see `mustFind` throws an `AssertionError` if the metric name/tags/type aren't found in the registry.

The previous `find` still exists. But since it is the less frequently used, it could probably use a different name, so `mustFind` is the easier to find.